inference-engine/src/transformations/include/transformations/op_conversions/gelu7_downgrade.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <transformations_visibility.hpp>
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+    class TRANSFORMATIONS_API Gelu7Downgrade;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief Gelu7Downgrade converts v7::Gelu6 operation to v2::Gelu unconditionally. This is done because only limited
+ * set of plugins support v7::Gelu which has an attribute specifying approximation mode. For other plugins the
```

Comment:
use the same name here and in the beginning: v7::Gelu or v7:;Gelu6 in both cases

Reply:
Fixed. Thanks.